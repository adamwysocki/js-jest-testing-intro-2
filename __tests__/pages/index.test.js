import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../../pages/index';

/** test the index page */
describe("Integration tests", () => {
    it('should match a snapshot', () => {
        const component = renderer.create(<Index />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
