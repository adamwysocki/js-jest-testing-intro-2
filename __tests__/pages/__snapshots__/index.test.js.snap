// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Integration tests should match a snapshot 1`] = `
<div
  className="container"
  style={
    Object {
      "marginTop": "150px",
    }
  }
>
  <h1>
    Hello!
  </h1>
  <form
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <input
        aria-describedby="nameHelp"
        className="form-control"
        id="name"
        onChange={[Function]}
        placeholder="Enter name ..."
        type="text"
        value=""
      />
    </div>
    <div
      className="form-check"
    >
      <input
        checked={false}
        className="form-check-input"
        id="espanol"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="form-check-label"
        htmlFor="espanol"
      >
        espa√±ol
      </label>
    </div>
  </form>
</div>
`;
